pyitlib/discrete_random_variable.py

```diff
@@ -4752,6 +4752,9 @@ def _sanitise_array_input(X, fill_value=-1):
     if X.size == 1 and X.ndim == 0:
         X = np.array((X, ))
 
+    if X.dtype.kind == "O":
```

Comment:
`None` will create a `dtype` of `Object` which sklearn does not like in the encoder transform.

Reply:
@DongchaoChen have you addressed this?

Reply:
> @DongchaoChen have you addressed this?

I only addressed this package build, @GeoffBarrett made this change as he explained above